pkg/server/handler/asset.go

```diff
@@ -133,49 +139,72 @@ func (h *AssetGetURLHandler) Handle(payload *router.Payload, response *router.Re
 
 // AssetUploadURLHandler receives and persists a file to be associated by Record.
 //
-// Example curl:
+// Example curl (PUT):
 //	curl -XPUT \
 //		-H 'X-Skygear-API-Key: apiKey' \
 //		-H 'Content-Type: text/plain' \
 //		--data-binary '@file.txt' \
 //		http://localhost:3000/files/filename
+//
+// Example curl (POST):
+//	curl -XPOST \
+//    -H "X-Skygear-API-Key: apiKey" \
+//    -F 'file=@file.txt' \
+//    http://localhost:3000/files/filename
+//
 type AssetUploadURLHandler struct {
 	AssetStore    skyAsset.Store   `inject:"AssetStore"`
 	AccessKey     router.Processor `preprocessor:"accesskey"`
 	DBConn        router.Processor `preprocessor:"dbconn"`
 	preprocessors []router.Processor
 }
 
+type assetUploadRequest struct {
```

Comment:
this struct does not belong here, move it elsewhere (above struct definition or below all functions defined for a struct